redash/query_runner/google_spreadsheets.py

```diff
@@ -147,6 +147,9 @@ def request(self, *args, **kwargs):
 
 
 class GoogleSpreadsheet(BaseQueryRunner):
+    def __init__(self, configuration):
+        super(GoogleSpreadsheet, self).__init__(configuration)
+        self.syntax = 'custom'
```

Comment:
Did you check that the UI doesn't blow with syntax names it doesn't recognize?

Reply:
Seems like this only affects query formatting. I tested it by changing `BaseQueryRunner`'s syntax to `custom` and noticed no explosions other than query formatting.